src/Live.php

```diff
@@ -93,6 +93,7 @@ public function callback()
         );
         $params = $this->addParams(array('client_id', 'client_secret', 'state'), $params);
 
+        $this->http->setFormUrlencoded();
```

Comment:
This breaks the `HttpClientInterface` contract. `$this->http` is never guaranteed to be the type of client you've changed in the other pull request (https://github.com/Claromentis/opauth/pull/2).